src/Constants.ts

```diff
@@ -430,6 +432,19 @@ export const ESTags = {
     ES_BODY: 'elasticsearch.body',
 };
 
+interface AMQPLIBType {
+    [key: string]: string;
+}
+
+export const AMQPTags: AMQPLIBType = {
```

Comment:
We should also have a tag for the message itself: `amqp.message`

Reply:
done.